.github/workflows/merge-main-to-develop-after-push.yml.OLD

```diff
@@ -0,0 +1,38 @@
+name: GitHub Actions - Merge modifications made on main to develop
```

Comment:
This file will not be used...
Keep it anyway?

Reply:
Yeah, I want to work on it in the following days. If it still is not necessary at all, I will delete it

Reply:
Okay 👍